src/Type/FileTypeMapper.php

```diff
@@ -63,6 +63,7 @@ private function createTypeMap(string $fileName): array
 			'#@var\s+\$[a-zA-Z0-9_]+\s+' . self::TYPE_PATTERN . '#',
 			'#@return\s+' . self::TYPE_PATTERN . '#',
 			'#@property(?:-read)?\s+' . self::TYPE_PATTERN . '\s+\$[a-zA-Z0-9_]+#',
+			'#@method\s+' . self::TYPE_PATTERN . '\s+[a-zA-Z0-9_]+\(.*\)#',
```

Comment:
You should bump the number in `v22` on line 40 because you're changing what type info is discovered in a file.